spec/rubocop/cop/style/sample_spec.rb

```diff
@@ -46,28 +52,33 @@
 
   it_behaves_like('accepts', 'sample')
   it_behaves_like('accepts', 'shuffle')
-  it_behaves_like('accepts', 'shuffle.at(2)')          # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle.at(2)')           # nil if coll.size < 3
   it_behaves_like('accepts', 'shuffle.at(foo)')
-  it_behaves_like('accepts', 'shuffle.slice(2)')       # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle.slice(3, 3)')    # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle.slice(2..3)')    # empty if coll.size < 3
+  it_behaves_like('accepts', 'shuffle.slice(2)')        # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle.slice(3, 3)')     # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle.slice(2..3)')     # empty if coll.size < 3
   it_behaves_like('accepts',
-                  'shuffle.slice(2..-3)')   # can't compute range size
+                  'shuffle.slice(2..-3)')               # can't compute range size
   it_behaves_like('accepts',
-                  'shuffle.slice(foo..3)')  # can't compute range size
-  it_behaves_like('accepts', 'shuffle.slice(-4..-3)')  # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle.slice(foo)')     # foo could be a Range
-  it_behaves_like('accepts', 'shuffle.slice(foo, 3)')  # nil if coll.size < foo
+                  'shuffle.slice(foo..3)')              # can't compute range size
+  it_behaves_like('accepts', 'shuffle.slice(-4..-3)')   # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle.slice(foo)')      # foo could be a Range
+  it_behaves_like('accepts', 'shuffle.slice(foo, 3)')   # nil if coll.size < foo
   it_behaves_like('accepts', 'shuffle.slice(foo..bar)')
   it_behaves_like('accepts', 'shuffle.slice(foo, bar)')
-  it_behaves_like('accepts', 'shuffle[2]')           # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle[3, 3]')        # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle[2..3]')        # empty if coll.size < 3
-  it_behaves_like('accepts', 'shuffle[2..-3]')       # can't compute range size
-  it_behaves_like('accepts', 'shuffle[foo..3]')      # can't compute range size
-  it_behaves_like('accepts', 'shuffle[-4..-3]')      # nil if coll.size < 3
-  it_behaves_like('accepts', 'shuffle[foo]')         # foo could be a Range
-  it_behaves_like('accepts', 'shuffle[foo, 3]')      # nil if coll.size < foo
+  it_behaves_like('accepts', 'shuffle[2]')              # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle[3, 3]')           # nil if coll.size < 3
+  it_behaves_like('accepts', 'shuffle[2..3]')           # empty if coll.size < 3
+  it_behaves_like('accepts', 'shuffle[2..-3]')          # can't compute range size
+  it_behaves_like('accepts', 'shuffle[foo..3]')         # can't compute range size
+  context 'Ruby >= 2.6', :ruby26 do
+    it_behaves_like('accepts', 'shuffle[3..]')            # can't compute range size
+    it_behaves_like('accepts', 'shuffle[3...]')           # can't compute range size
```

Comment:
It would be nice to align these :-)

```suggestion
    it_behaves_like('accepts', 'shuffle[3..]')          # can't compute range size
    it_behaves_like('accepts', 'shuffle[3...]')         # can't compute range size
```

Reply:
✅